ui/src/org/pentaho/di/ui/core/widget/TextVar.java

```diff
@@ -153,7 +153,7 @@ public void setInsertTextInterface( InsertTextInterface insertTextInterface ) {
     this.insertTextInterface = insertTextInterface;
   }
 
-  private ModifyListener getModifyListenerTooltipText( final Text textField ) {
+  protected ModifyListener getModifyListenerTooltipText( final Text textField ) {
     return new ModifyListener() {
       public void modifyText( ModifyEvent e ) {
         if ( textField.getEchoChar() == '\0' ) { // Can't show passwords ;-)
```

Comment:
If the PasswordTextVar class is used, the getModifyListenerTooltipText() method is overridden, so that only the ToolTip text is shown.  I've change the overridden class to be a NO-OP, and added if statements in the TextVar class to avoid NPE issues.